src/hugr/hugrmut.rs

```diff
@@ -216,10 +216,7 @@ impl InsertionResult {
 }
 
 /// Impl for non-wrapped Hugrs. Overwrites the recursive default-impls to directly use the hugr.
-impl<T> HugrMut for T
-where
-    T: HugrView + AsMut<Hugr>,
-{
+impl<T: HugrView + AsMut<Hugr>> HugrMut for T {
```

Comment:
Hmmm, these were a bit drive-by, but they make searching for these quasi-blanket impls much easier. (A one-line search hit `impl<T> HugrMut for T` is really not very informative!)